### ISSUE TITLE
Possibility of terminal support?

### ISSUE THREAD

**Issue by Graydenn** [arcusmaximus/YTSubConverter#29]
Hello,

I'm trying to archive a bunch of YouTube videos with stylized subtitles. 

Your tool is extremely useful, especially with the newer conversion feature, however downloading the sub file from YouTube and then manually running it through the GUI has ended up being the most time consuming step in this process.

I have no idea how hard this would be to implement (or make) but I was wondering if you (or someone else) could make a command line tool to convert the YouTube sub file to ASS in a way that would allow it to be used alongside youtube-dl.

That's all. Thanks for your work!

**Comment by arcusmaximus** [comment]
You might be surprised to learn that YTSubConverter has command line support already! It's mentioned in the "Reverse conversion" section of the documentation. However, the fact that you didn't find it indicates that the documentation wasn't clear enough, so I added an extra "Command line" section with some more detailed information.

I also added back the explanation about downloading .srv3 subtitles with youtube-dl to the "Reverse conversion" section. (It was there originally, but I removed it when youtube-dl got DMCA'd and forgot to put it back afterwards)
